Resources/Locale/en-US/GPS/handheld-gps.ftl

```diff
@@ -1 +1,3 @@
-handheld-gps-coordinates-title = Coords: {$coordinates}
+handheld-gps-coordinates-title =
+    Coords:
+    {$coordinates}
```

Comment:
Fluent does not have character escapes like that. It will interpret `\n` literally, and rely on something else to convert it to a newline. This seems like borderline abuse of Fluent if you ask me.

I could write `Coords:{"\n"}{$coordinates}`, but why? The entire point of Fluent is that, outside `{}`, what you see is what you get. :)

Reply:
Mmmm. That was definitely a design choice on flient's part...

Well, then the pr looks all good to me, then.